helm_deploy/licences/templates/_envs.tpl

```diff
@@ -153,4 +153,7 @@ env:
 
   - name: LICENCE_RISK_MANAGEMENT_VERSION
     value: {{ .Values.env.LICENCE_RISK_MANAGEMENT_VERSION | quote }}
+
+  - name: CURFEW_ADDRESS_REVIEW_VERSION
+    value: {{ .Values.env.CURFEW_ADDRESS_REVIEW_VERSION | quote }}
```

Comment:
Should this be `CURFEW_ADDRESS_VERSION` rather than review specific? I think that makes sense as it's the version of the curfew address object in the licence?

Reply:
I thought CURFEW_ADDRESS_REVIEW_VERSION was more appropriate as the db object is curfewAddressReview

Reply:
Ah yes - apologies I was looking at the `curfewAddress` section and getting confused - typical HDC confusion - ignore that point from me then 😄